cypress/integration/metaSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('The example page with imgix meta tags', () => {
+  it('successfully loads', () => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+    // .should('equal', 'ix:useHttps');
+    cy.get('meta').each(($el) => {
+      const property = $el.attr('property');
+      if (!!property) {
+        expect(property).to.include('ix');
+      }
+    });
+  });
+  it('successfully applies imgix paramenters to the source url', () => {
```

Comment:
This test is brittle for a few reasons:

- if the rendered url changes the order of its query parameters then the test breaks
- if the package version changes the test breaks

I would recommend using something like the `URL` API and checking that the parameters are applied.

---

From a different perspective, I'm not sure how valuable this is as a test since we already have tests for parameters being set. An idea could be to simply test if the image loads, since this is then improving on our existing test suite. An example of a test like this can be seen in other Cypress tests we have, such as this one: https://github.com/imgix/gatsby/blob/main/test/dev-and-e2e/cypress/e2e/gatsby-transform-node.js